util/http.cc

```diff
@@ -5,9 +5,12 @@
 #include "memory.h"
 #include "strings.h"
 
+#include <algorithm>
 #include <curl/curl.h>
+#include <curl/multi.h>
 #include <rapidjson/stringbuffer.h>
 #include <rapidjson/writer.h>
+#include <sstream>
```

Comment:
Off hand I don't see where anything from this was added.

Reply:
Thanks. It was a left over from some debugging I was doing.